roles/openshift_master/tasks/main.yml

```diff
@@ -218,6 +218,23 @@
   - restart master api
   - restart master controllers
 
+- name: Check for credentials file for registry auth
+  stat:
+    path: "{{ registry_auth_credentials_path }}"
+  when:
+  - registry_auth_user is defined
+  - openshift_examples_modify_imagestreams | bool
+  register: master_registry_auth_credentials_stat
+
+- name: Create credentials for registry auth
+  command: "docker --config={{ registry_auth_credentials_path }} login -u {{ registry_auth_user }} -p {{ registry_auth_token }} {{ oreg_host }}"
```

Comment:
It is better to mention that in inventory/byo/hosts.ose.example, give some example to show user how to define registry_auth_user and registry_auth_token.